Sources/TZSettingsView/TZSettingsList/TZSettingsSectionView.swift

```diff
@@ -44,6 +44,10 @@ struct TZSettingsSectionView: View {
                             case .action(let action):
                                 TZSettingsActionCellView(configuration: cell.cellConfiguration,
                                                           action: action)
+
```

Comment:
What is the difference between action and descriptionWithAction, I can not notice any difference